firmware/vcv_plugin/export/src/module_widget_adaptor.hh

```diff
@@ -156,6 +170,7 @@ struct ModuleWidgetAdaptor {
 				el.coords = Coords::TopLeft;
 				el.short_name = name;
 				el.long_name = name;
+				el.visible = widget->visible;
```

Comment:
Ah forgot to remove, but actually would you be willing to add `bool visible` to BaseElement? Currently visibility status not respected (useful for hidden params).